lib/Platform/Logging.cpp

```diff
@@ -20,13 +24,32 @@
 
 namespace hermes {
 
+#if defined(_MSC_VER)
+// Define the GUID to use in TraceLoggingProviderRegister
+// {0x5bf36bca, 0x6f36, 0x44a7, {0x83, 0x6f, 0x4f, 0x41, 0xad, 0x75, 0x86,
+// 0x32}}
```

Comment:
What's the source of this GUID? Is it something meaningful?
Does it have some format like 123456abc-1234-1234-1234567 that's more human readable and copyable?

Reply:
Hi Dulin .. This is a random generated GUID, which globally identifies our "Facebook.hermes" ETW provider uniquely. The same GUID should be provided in the WPR profile (utils\tracing\win\hermes.wprp) to be used when starting and stopping the session using WPR .. 

My understanding is that, this format must be used when setting GUID variable in C/C++ source as the structure looks like, (There are some macros to work with registry format, declspect format etc. but they are more blackbox for me .. )

[REF: Windows Kits\10\Include\10.0.19041.0\shared\guiddef.h]
typedef struct _GUID {
  unsigned long  Data1;
  unsigned short Data2;
  unsigned short Data3;
  unsigned char  Data4[8];
} GUID;